dags/fivetran_acoustic.py

```diff
@@ -163,8 +168,15 @@ def _generate_acoustic_report(conn_id: str, report_type: str, config: Dict[Any,
             poke_interval=30
         )
 
-        # TODO: trigger corresponding BQETL Dag
-        # TODO: Wait for that Dag to finish
+        downstream_dag = dag_id.replace("fivetran", "bqetl")
+
+        trigger_dependent_dag_and_wait = TriggerDagRunOperator(
+            task_id="trigger_dependent_bqetl_process",
+            trigger_dag_id=downstream_dag,
+            wait_for_completion=True,
+            execution_date="{{ ds }}",
+            reset_dag_run=True,
+        )
```

Comment:
I figured since the BQETL DAG is designed to truncate the source table after populating the derived table, I figured this DAG should not continue until that process has succeeded.

Reply:
open to having a chat, for now will merge since this is not production yet anyways, but I can proceed with testing.